src/modules/twap/hooks/useCreateTwapOrder.ts

```diff
@@ -41,11 +44,22 @@ export function useCreateTwapOrder() {
       outputAmount: outputCurrencyAmount,
     }
 
+    const twapFlowAnalyticsContext: SwapFlowAnalyticsContext = {
+      account,
+      recipient: twapOrder.receiver,
+      recipientAddress: twapOrder.receiver,
+      marketLabel: [inputCurrencyAmount.currency.symbol, outputCurrencyAmount.currency.symbol].join(','),
+      // TODO: change to TWAP?
```

Comment:
The class the suborders will be placed as is still the same, but you are right that here this doesn't apply.
We should then have this property be unlinked from OrderClass order property and have a property exclusive for analytics.

Can you create that?